docs/userguides/contracts.md

```diff
@@ -16,7 +21,17 @@ dev = accounts.load("dev")
 contract = project.MyContract.deploy(sender=dev)
 ```
 
-## From Contract
+## From Project Contract Address
+
+You can also use the [at() method](../methoddocs/contracts.html#ape.contracts.base.ContractContainer.at) method from the same top-level project manager when you know the address of an already-deployed contract:
+
+```python
+from ape import project
+
```

Comment:
do you need a load an account that owns the address first to do any owner privilege methods on the contract? 
`dev = accounts.load('dev')`
`contract.mint("vitalik.eth", sender=dev)`

Should we make a side note for that?

Reply:
there is a section at the bottom of this guide about contract interaction where we show how to make transactions from accounts.

this part is only about getting the contract instance

Reply:
> do you need a load an account that owns the address first to do any owner privilege methods on the contract? 

No you don't